gvsbuild/utils/base_project.py

```diff
@@ -108,7 +109,7 @@ def __init__(self, name, **kwargs):
         self.extra_env = {}
         self.tag = None
         self.repo_url = None
-        self.archive_filename = None
+        self.extra_opts = None
```

Comment:
Hi @DjLegolas, did you mean to delete `self.archive_filename` here? Anything else we need to do to get this merged?